### PR TITLE
meetup guide (talks format)

### PR DESCRIPTION
**Pull request by booyaa**

### CODE REVIEW

File: standards/MEETUP_TALKS.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Ensure that all speakers are aware of this limitation. 
+
+The recommended approach advise the speaker of time left, is to have warnings at 5min intervals starting at 15mins remaining of talk duration. Tip: you can tell the speaker through the use of cards with the time remaining or verbally. The latter is useful to remember since you won't have a mic so the video recording if you have one will not pick up the prompts.
+
+Also don't forget to time keep the Q&A section!
+
+## Hints and tips
+
+- **Booking venue**: it's often easier to pre-book in quarters i.e. January to March, April to June, etc.
+- **Beg, steal, borrow**: if you attend other techical meetups, try booking their venue.
```
Comment (Manishearth) [inline standards/MEETUP_TALKS.md]: Often companies that use Rust will provide a venue for free.

File: standards/MEETUP_TALKS.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# How to organise a talk based meet up
+
+## Introduction
+
+The purpose of this document is to provide a checklist and timeline for organising talk based meetups. This is based around the process used by the [London Rust User Group][link_lrug].
+
+The expected audience for this document are people who are new to setting up meetups or organising this type of format.
+
+Where relevant I will cite actual examples using the names of services or organisations (Skillsmatter, meetup.com, etc) that we currently use at the London user group.
+
```
Comment (Manishearth) [inline standards/MEETUP_TALKS.md]: Also ask them to fill in https://docs.google.com/forms/d/e/1FAIpQLSf52YXGhqBaHtCXtVna4iHYMK7IQaTqUW6V-ztsZC8C2TBInQ/viewform and join https://groups.google.com/forum/#!forum/rust-meetup-organizers  so they have a place to ask questions?

File: standards/MEETUP_TALKS.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+The recommended approach advise the speaker of time left, is to have warnings at 5min intervals starting at 15mins remaining of talk duration. Tip: you can tell the speaker through the use of cards with the time remaining or verbally. The latter is useful to remember since you won't have a mic so the video recording if you have one will not pick up the prompts.
+
+Also don't forget to time keep the Q&A section too!
+
+## Hints and tips
+
+- **Automate**: Simplify your workflow for organising meetups by making use of [issue templates][link_gh_templates]. Templates can generate a standardised checklist. The use of [query parameters][link_gh_query_params] can also add labels and a title. The London template can be found [here][link_lon_talks_template].
+- **Finding a venue**: Often companies that use Rust will provide a venue for free. There's a list of companies on the [Rust front page][link_rust_friends], some of these companies might be near to your city.
+- **Booking venue**: it's often easier to pre-book in quarters i.e. January to March, April to June, etc.
+- **Beg, steal, borrow**: if you attend other techical meetups, try booking their venue.
```
Comment (badboy) [inline standards/MEETUP_TALKS.md]: Can we change the wording here? I don't think "beg" and "steal" are the right words here.

File: standards/MEETUP_TALKS.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+- **Speakers**: if you're struggling to find local speakers, why not try a remote talk. These are trickier to setup, but can sometimes give you the access to speakers who wouldn't ordinarily be available because of your location.
+- **#rustlang**: The Rust [@rustlang][link_rust_twitter] twitter account will usually retweet any tweets with the #rustlang has tag, so use this to advertise your meetup.
```
Comment (badboy) [inline standards/MEETUP_TALKS.md]:
```patch
- has tag
+ hash tag
```

**Comment by celaus** [comment]
Hey @booyaa - Thank you for the guide! We'll merge it shortly!

**Pull request merged**